scripts/create-new-actor.sh

```diff
@@ -322,24 +322,23 @@ create_impl                  "$uuid_tag" "$actor_name"
 create_test                  "$actor_name"
 create_workload_yml          "$actor_name"
 
+actor_name_lowercase=$(echo "$actor_name" | tr '[:upper:]' '[:lower:]');
```

Comment:
why do we lowercase the actor name? keeping it consistent would get us out of having multiple case conventions to worry about.

Reply:
Hm, catch2's cmake integration does it for you: https://github.com/mongodb/genny/blob/916a2c20a3eb7813dbbc9595418a381099e09c89/cmake/ParseAndAddCatchTests.cmake#L144

No idea why.

I commented out the above line to have the tags' casing be consistent catch2 and ctest.